yaksh/views.py

```diff
@@ -117,14 +117,25 @@ def user_logout(request):
 def quizlist_user(request, enrolled=None):
     """Show All Quizzes that is available to logged-in user."""
     user = request.user
-    if enrolled is not None:
+    ci = RequestContext(request)
+
+    if request.method == "POST":
+        course_code = request.POST.get('course_code')
+        hidden_courses = Course.objects.get_hidden_courses(code=course_code)
+        courses = hidden_courses if hidden_courses else None
+        title = 'Search'
+
+    elif enrolled is not None:
```

Comment:
Why do we have the search bar on the enrolled courses page? Having the search bar on two different pages is a bit unnecessary.